src/sentry/integrations/slack/utils.py

```diff
@@ -173,6 +180,20 @@ def build_rule_url(rule, group, project):
     return absolute_uri(rule_url)
 
 
+def build_upgrade_notice_attachment(group, integration):
+    org_slug = group.organization.slug
+    url = absolute_uri(u"/settings/{}/integrations/slack/{}/".format(org_slug, integration.id))
```

Comment:
Will we get metrics through this on our Amplitude referrer graphs?

Reply:
@manuzope good callout! I need to add that.